cli/replicaset/cmd/lua/upgrade.lua

```diff
@@ -0,0 +1,11 @@
+local ok, err
+ok, err = pcall(box.schema.upgrade)
```

Comment:
The same, but shortest.
```suggestion
local ok, err = pcall(box.schema.upgrade)
```

Reply:
Done.